_docs-2/getting-started/quickstart.md

```diff
@@ -40,6 +40,16 @@ For convenience, consider adding `accumulo-{{ page.latest_release }}/bin/` to yo
 Accumulo requires running [Zookeeper] and [HDFS] instances which should be set up
 before configuring Accumulo.
 
+**Important note:** As of version 3.0.0, HDFS supports [Erasure Coding] (EC) in addition to
```

Comment:
Could shorten this, just let people know they may loose data, but not get into the details.  Maybe something like the following 

```
**Important note:** If using [Erasure Coding] (EC) , data loss will occur unless it is configured
properly for Accumulo. Please see the [Erasure Coding guide][ec-guide] for more information.
```

Reply:
sounds reasonable, I'll fix that.

There will be some changes to the EC guide soon...I just found a gotcha with the new bulk import not playing nice with EC.  Seems the default behavior is 8*<num_core> threads for doing the scan of the import files, but with EC each thread is going to create a threaded stripe reader, so there's an explosion of threads and strange out-of-memory errors abound.  Took me all day to track that one down.

Reply:
> so there's an explosion of threads and strange out-of-memory errors

I have run into situation where you get an OOME when trying to create a new thread because of the stack space limitations.   Could experiment with `-Xss` jvm option.

Reply:
Would you happen to know if the OOME problem happens when not using EC?  How many files were you bulk importing when this happened?

Reply:
It does not occur when EC is not in use (default replication 3).  There were 1024 files to bulk load.  I don't recall exactly how big they were, but IIRC each file should map to a single tablet.  Won't be at work this week to check.